model/project_ref.go

```diff
@@ -1977,6 +1981,19 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 	var err error
 	switch section {
 	case ProjectPageGeneralSection:
+		if utility.FromBoolPtr(p.RepotrackerDisabled) && p.RemotePath == "" {
```

Comment:
we can use p.IsRepotrackerDisabled() 
Also we'd want to check if it's enabled here, not disabled, right?